Shared/R4/resources/Questionnaire-R4-EncounterItems.json

```diff
@@ -0,0 +1,78 @@
+{
+  "resourceType": "Questionnaire",
+  "id": "encounter-items",
+  "name": "Encounter Module",
+  "meta": {
+    "profile": [
+      "http://hl7.org/fhir/StructureDefinition/cqf-questionnaire",
+      "http://hl7.org/fhir/us/davinci-dtr/StructureDefinition/dtr-questionnaire-r4"
+    ]
+  },
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicClinicalInfo-prepopulation"
+    }
+  ],
+  "status": "draft",
+  "item": [
+    {
+      "linkId": "ENC.1",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+          "valueExpression": {
+            "language": "text/cql",
+            "expression": "\"BasicClinicalInfoPrepopulation\".Today"
```

Comment:
Most of the "Date of encounter" cases are not used in F2F Encounter form. Therefore, we should not assume it is "Today". The best way to retrieve it would be get from DeviceRequest/ServiceRequest referenced encounter, then find the date.

Reply:
This subQ is used by F2F and Progressive Note only.

Reply:
For ProgressNote, is it supposed to Today? Some of the order form also has F2F date